### PR TITLE
Initial PR for contributing my ortho_4x12 community layout.

### PR DESCRIPTION
**Pull request by jdmcfad**
The `keymap.c` has my original implementation of randomized clicky beeps and also has an implementation of randomized backlight flickering.  See the `readme.md` for other layout details -- the layout is not very different from the default Planck keymap, but this 4x12 layout has been tested with three boards and also builds for the Planck rev 6 (I'm patiently waiting to test).

I've had this fork out since 4/4/2017.  :)

### CODE REVIEW

File: layouts/community/ortho_4x12/mindsound/config.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline layouts/community/ortho_4x12/mindsound/config.h]: Could you replace the ifndef and the define here with just `#pragma once`, and remove the endif at the end?

File: layouts/community/ortho_4x12/mindsound/clicky.h [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#ifndef CLICKY_H
```
Comment (drashna) [inline layouts/community/ortho_4x12/mindsound/clicky.h]: Could you replace the ifndef and the define here with just `#pragma once`, and remove the endif at the end?

File: layouts/community/ortho_4x12/mindsound/flicker.h [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#ifndef FLICKER_H
```
Comment (drashna) [inline layouts/community/ortho_4x12/mindsound/flicker.h]: Could you replace the ifndef and the define here with just `#pragma once`, and remove the endif at the end?

File: layouts/community/ortho_4x12/mindsound/keymap.c [hunk 4]
```diff
@@ -0,0 +1,327 @@
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+// this is the style you want to emulate.
+
+#include QMK_KEYBOARD_H
+
+#include "action_layer.h"
```
Comment (drashna) [inline layouts/community/ortho_4x12/mindsound/keymap.c]: Most of these includes aren't needed.  The clicky, and flicker ones are probably the only ones needed here.

File: layouts/community/ortho_4x12/mindsound/config.h (hunk 1 above)
Comment (jdmcfad) [reply layouts/community/ortho_4x12/mindsound/config.h]: aye aye

File: layouts/community/ortho_4x12/mindsound/clicky.h (hunk 2 above)
Comment (jdmcfad) [reply layouts/community/ortho_4x12/mindsound/clicky.h]: aye aye

File: layouts/community/ortho_4x12/mindsound/flicker.h (hunk 3 above)
Comment (jdmcfad) [reply layouts/community/ortho_4x12/mindsound/flicker.h]: aye aye

File: layouts/community/ortho_4x12/mindsound/keymap.c (hunk 4 above)
Comment (jdmcfad) [reply layouts/community/ortho_4x12/mindsound/keymap.c]: Hm, I needed QMK_KEYBOARD_H or my branch wouldn't compile.

Comment (drashna) [reply layouts/community/ortho_4x12/mindsound/keymap.c]: Yes, that's the one that is needed. :)
The rest most likely not.

**Comment by jdmcfad** [comment]
@drashna I removed `#include "audio.h"` as well -- it was making VS code happy but wasn't necessary for compilation.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**